shared/js/background/abp-lists.es6.js

```diff
@@ -1,16 +1,11 @@
 /*
  * Load the abp-filter-parser node module and
  * pre-process the easylists.
- *
- * This will be browserifyed and turned into abp.js by running 'grunt'
  */
 const abp = require('abp-filter-parser')
-const deepFreeze = require('deep-freeze')
 const constants = require('../../data/constants')
-const defaultSettings = require('../../data/defaultSettings')
 const surrogates = require('./surrogates.es6')
 const settings = require('./settings.es6')
-const ATB = require('./atb.es6')
```

Comment:
A lot of the variable declarations I've removed were marked as not used anywhere.